cypress/support/commands.js

```diff
@@ -10,7 +10,27 @@
 //
 //
 // -- This is a parent command --
-// Cypress.Commands.add("login", (email, password) => { ... })
+Cypress.Commands.add("login", (email, password) => {
+
+  // it is ok for the email to be visible in the Command Log
+  expect(email, "email was set").to.be.a("string").and.not.be.empty;
+  // but the password value should not be shown
+  if (typeof password !== "string" || !password) {
+    throw new Error("Missing password value, set using CYPRESS_password= or in cypress env file...");
+  }
+
+  cy.get('input[name="emaial"]').type(email);
+
+  cy.get("[name=passwd]")
+    .type(password, { log: false })
+    .should($password => {
```

Comment:
I'm not sure this test would ever fail as it's testing password against password (which you just typed). If not, remove it since it's not useful.

Reply:
Removed